dbt/include/databricks/macros/python.sql

```diff
@@ -0,0 +1,77 @@
+{% macro databricks__py_write_table(compiled_code, target_relation) %}
```

Comment:
This file is called `python.sql` but it writes Python syntax. Can we change its extension to `.py`? Or is the `.sql` file-type required by dbt?

Reply:
the .sql file is a dbt standard (even for python code), but I haven't tried it as a different file extension.  Personally, I would have named the files something like adapter.sql.template or adapter.sql.jinja.  I'll see what happens if I use a different extension.

Reply:
Did not work with me changing the extension...without understanding a lot more about how the macro engine works, I think we have to stick with .sql.